lib/script.js

```diff
@@ -0,0 +1,70 @@
+/**** To change time from UNIX to normal ***/
+// var time = new Date(timestamp*1000);
+
+var weather_data = [];
+
+$(document).ready(function() {
+    ShowCurrentTime();
+    getWeather("denver", "co");
+    pauseThenLoadWeather();
+})
+
+const delay = ms => new Promise(res => setTimeout(res, ms));
+const pauseThenLoadWeather = async () => {
+    document.getElementById('fahrenheit-symbol').innerHTML = ""
+    await delay(2000);
+    document.getElementById('fahrenheit-symbol').innerHTML = "F";
+    displayCurrentWeather();
+    document.querySelector('.lds-roller').style.display = 'none';
+  };
```

Comment:
This pause function is great! It's the perfect place to implement this concept. Well done!